examples/auth/types.ts

```diff
@@ -9,4 +10,8 @@ declare module "blitz" {
     userId: User["id"]
     views?: number
   }
+  // export type IsAuthorized = typeof simpleRolesIsAuthorized
+  export interface Authorize extends DefaultAuthorize {
+    (roleOrRoles?: string | string[], options?: {if?: boolean}): boolean
+  }
 }
```

Comment:
User provided type override (same approach as `PublicData` which works fantastically)

Reply:
The only thing the user really needs to modify is the arguments to the function. The return type is always fixed. So maybe there's another way to structure this that is better.